src/main/java/uk/gov/companieshouse/api/accounts/validation/AccountsValidator.java

```diff
@@ -1,27 +1,65 @@
 package uk.gov.companieshouse.api.accounts.validation;
 
+import javax.servlet.http.HttpServletRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.links.CompanyAccountLinkType;
+import uk.gov.companieshouse.api.accounts.model.rest.CompanyAccount;
+import uk.gov.companieshouse.api.accounts.model.rest.SmallFull;
 import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.CompanyAccountService;
+import uk.gov.companieshouse.api.accounts.service.CompanyService;
+import uk.gov.companieshouse.api.accounts.service.impl.SmallFullService;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
 import uk.gov.companieshouse.api.accounts.validation.transactionclosure.CurrentPeriodTnClosureValidator;
+import uk.gov.companieshouse.api.accounts.validation.transactionclosure.PreviousPeriodTnClosureValidator;
 import uk.gov.companieshouse.api.model.transaction.Transaction;
 
-import javax.servlet.http.HttpServletRequest;
-
 @Component
-public class AccountsValidator {
+public class AccountsValidator extends BaseValidator {
+
+    private static final String SMALL_FULL_PATH = "$company_accounts.small_full";
+
+    private CompanyAccountService companyAccountService;
```

Comment:
These fields could be final

Reply:
Ok 👍

Reply:
Just a thought, should the path be "$.company_accounts.small_full" or should it be what it currently is? "$company_accounts.small_full"

Genuine question :)